eo-maven-plugin/src/test/java/org/eolang/maven/ResolveMojoIT.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.maven;
+
+import com.jcabi.manifests.Manifests;
+import com.yegor256.MayBeSlow;
+import com.yegor256.Mktmp;
+import com.yegor256.MktmpResolver;
+import com.yegor256.WeAreOnline;
+import com.yegor256.farea.Farea;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Integration tests for {@link ResolveMojo}.
+ *
+ * @since 0.52
+ */
+@SuppressWarnings({"JTCOP.RuleAllTestsHaveProductionClass", "JTCOP.RuleNotContainsTestWord"})
+@ExtendWith({WeAreOnline.class, MktmpResolver.class, MayBeSlow.class})
+final class ResolveMojoIT {
+    @Test
+    void resolvesJarFile(@Mktmp final Path temp) throws IOException {
+        new Farea(temp).together(
+            f -> {
+                ResolveMojoIT.configureFarea(f);
+                f.exec("process-classes");
+                MatcherAssert.assertThat(
+                    "the jar file was resolved and unpacked",
+                    f.files().file(
+                        String.format(
+                            "target/eo/%s/org.eolang/eo-runtime/-/0.39.0/org/eolang/Phi.class",
```

Comment:
@Suban05 let's move `"0.39.0"` to variable and use it in `configureFarea` function call and here in `String.format`. It would be less magic in the test